### DISCUSSION TITLE
Renovação

### DISCUSSION THREAD

**Discussion by Bigua** [apyb/comunidade#155] [Pergunta]
Olár, chegou por email o link de renovação mas ao clicar nele só tem a opção associar e não tem renovar. É assim mesmo?

**Comment by naanadr** [comment]
Sim @Bigua , porque agora estamos fazendo a cobrança em um outra ferramenta.

Você vai precisar se cadastrar no sistema da Stripe, que ficará agora fazendo a cobrança recorrente da anuidade.

**Reply by rougeth** [reply]
O [e-mail que enviamos](https://github.com/apyb/associados/blob/c3717d0d2e07a55b7eed78662b9f963b20ff6d41/app/payment/templates/payment/valid_until_email.txt) com a notificação para renovação não tem nenhuma informação sobre a mudança de plataforma. E se o @Bigua recebeu esse primeiro e-mail, quer dizer que vamos enviar [outros 4](https://github.com/apyb/associados/blob/c3717d0d2e07a55b7eed78662b9f963b20ff6d41/app/payment/management/commands/renewal_alert.py#L18C1-L18C53) com a mesma notificação, mesmo que ele já tenha renovado via Stripe.

**Reply by naanadr** [reply]
No e-mail consta as seguintes informações:

```
Caso você já tenha efetuado o pagamento pedimos que desconsidere esse aviso.

[...]

PS. O sistema de controle de associados está em fase de implantação. Caso encontre algum problema pedimos a gentileza de nos avisar [...]
```

Além delas, também comunicamos aos associados sobre a mudança de plataforma nas redes sociais, por e-mail, no site dos associados e no site da APyB. 

Mas de toda forma, posso mudar nos próximos dias o conteúdo do e-mail linkando para o comunicado oficial e esclarecendo melhor esses pontos.
